calico/felix/fetcd.py

```diff
@@ -207,6 +212,38 @@ def _periodically_report_status(self):
                 sleep_time = interval + jitter
                 gevent.sleep(sleep_time)
 
+    @logging_exceptions
+    def _periodically_version_signal(self):
```

Comment:
I guess one advantage you get of putting it here is that you could peak at `len(self.ipv4_by_hostname)` to get the cluster size.  However, that'd be a bit of a hack because we'd be accessing that dict concurrently from multiple threads.